lib/config.coffee

```diff
@@ -181,113 +179,3 @@ exports.validate = (config) ->
 
 	if disallowedProperty?
 		throw new Error("Validation: #{disallowedProperty} not recognized")
-
-getApplicationConfig = (application, options = {}) ->
```

Comment:
Why do you remove these?

Reply:
They're not used anywhere any more, and removing them means we can drop a load of dependencies